README.adoc

```diff
@@ -72,9 +72,11 @@ ncu
 
 This project includes GitHub action workflows to build and publish the book (PDF, EPUB, HTML). To trigger this, you have two options:
 
-* Use GitHub's UI to create the release from your book repository
 * Run the following script: `npm run release`
+* Use GitHub's UI to create the release from your book repository
 
 Once completed, the publish workflow will attach the generated files to the release page.
 
-Note that for the second option (i.e., the release npm script), you'll need to make sure that a GITHUB_TOKEN environment variable is available, as explained in [release-it's documentation](https://github.com/release-it/release-it/blob/master/docs/environment-variables.md).
+The benefit with the release npm script is that the version will be updated automatically in the "package.json" and "version.txt" files. The version will be reflected in the generated files because the version in version.txt is used as the gradle project version when building the book.
```

Comment:
```suggestion
The benefit of the release npm script is that the version will be updated automatically in the `package.json` and `version.txt` files. The version will be reflected in the generated files because the version in `version.txt` is used as the Gradle project version when building the book.
```